lib/provider-metadata.coffee

```diff
@@ -11,6 +11,14 @@ class ProviderMetadata
     if providerBlacklist = @provider.providerblacklist?['autocomplete-plus-fuzzyprovider']
       @disableDefaultProviderSelectors = Selector.create(providerBlacklist)
 
+  matchesEditor: (editor) ->
+    if @provider.textEditorSelectors?
+      atom.views.getView(editor).matches(
+          Array.from(@provider.textEditorSelectors).join(', '))
```

Comment:
Hmm, I thought about that too. I kept it a variable to be similar to `selector` but requiring it to be a array-like as part of the API might be a bit too specific. I changed it to a method returning a single selector.